bundle/manifests/submariner.clusterserviceversion.yaml

```diff
@@ -94,6 +103,15 @@ spec:
     ### Prerequisites
     Submariner has a few requirements to get started, all are documented [here](https://submariner.io/getting-started/quickstart/kind/#prerequisites).
 
+    If running on an OpenShift cluster, perform the following steps to bind the Submariner service accounts to the **privileged** SCC.
+
+    ```shell
+    oc adm policy add-scc-to-user privileged system:serviceaccount:submariner-operator:submariner-gateway
+    oc adm policy add-scc-to-user privileged system:serviceaccount:submariner-operator:submariner-routeagent
+    oc adm policy add-scc-to-user privileged system:serviceaccount:submariner-operator:submariner-globalnet
+    oc adm policy add-scc-to-user privileged system:serviceaccount:submariner-operator:submariner-lighthouse-coredns
+    ```
+
```

Comment:
Can we move these under the suggested "#### Deployment with the operator (without subctl)" ?  as those wouldn't apply to subctl?